prawns/governance/src/handlers/democracy.Started.event.ts

```diff
@@ -14,7 +14,7 @@ export default (network: SubstrateNetwork) =>
     const date = new Date(ctx.block.timestamp);
     const event = getDemocracyStartedEvent(ctx, network);
 
-    const democracyProposal = await substrateDemocracyProposalRepository.getByTabledAtBlock(ctx, blockNumber);
+    const democracyProposal = await substrateDemocracyProposalRepository.getByTabledAtBlock(ctx, network, blockNumber);
```

Comment:
Curiosity: Did you find this is clashing between networks?

Reply:
Its unlikely that they would ever clash given how few they are... just noticed none of them were filtering by network and fixed them wholesale :)